src/ios/ListLibraryItems.m

```diff
@@ -171,6 +171,11 @@ - (void)uploadItem:(CDVInvokedUrlCommand *)command {
                             
                             for(NSString * header in [headers allKeys]) {
                                 [headers setObject:[self getMimeTypeFromPath:resource.originalFilename] forKey:@"Content-Type"];
+                                // Adding creation date of asset to headers
+                                NSDateFormatter * df = [[NSDateFormatter alloc] init];
```

Comment:
what does `df` stand for?

Reply:
`df` stands for DateFormatter, it formats correctly the date, matching to server's date format.

Reply:
Maybe we can use a more explicit name? Like `creationDate` or something similar?

Reply:
In fact, we create a "dateformater" object and then we ask to this formater to... format the creationDate in the right format. So `df` is a good name I think ;). (see

``` [request setValue:[df stringFromDate:[asset creationDate]] forHTTPHeaderField:@"Date"];```
)

Reply:
Ah I see, alright!